homeassistant/components/habitica/calendar.py

```diff
@@ -225,3 +229,156 @@ def extra_state_attributes(self) -> dict[str, bool | None] | None:
         return {
             "yesterdaily": self.event.start < self.today.date() if self.event else None
         }
+
+
+class HabiticaTodoRemindersCalendarEntity(HabiticaCalendarEntity):
+    """Habitica to-do reminders calendar entity."""
+
+    entity_description = CalendarEntityDescription(
+        key=HabiticaCalendar.TODO_REMINDERS,
+        translation_key=HabiticaCalendar.TODO_REMINDERS,
+    )
+
+    def reminders(
+        self, start_date: datetime, end_date: datetime | None = None
+    ) -> list[CalendarEvent]:
+        """Reminders for todos."""
+
+        events = []
+
+        for task in self.coordinator.data.tasks:
+            if not (task["type"] == HabiticaTaskType.TODO and not task["completed"]):
```

Comment:
the double not is a little hard to follow. not todo or not completed might be easier to understand.